packages/std-client/src/utils.ts

```diff
@@ -56,9 +56,9 @@ export function getGameConfig(
   gameConfigComponent: Component<{ startTime: Type.String; turnLength: Type.String; actionCooldownLength: Type.String }>
 ) {
   const godEntityIndex = world.entityToIndex.get(GodID);
-  if (!godEntityIndex) return null;
+  if (!godEntityIndex) return;
```

Comment:
```suggestion
  if (godEntityIndex == null) return;
```
otherwise this fails if god happens to be at index 0